plugins/dell-kestrel/fu-dell-kestrel-dpmux.c

```diff
@@ -87,6 +87,7 @@ fu_dell_kestrel_dpmux_init(FuDellKestrelDpmux *self)
 	fu_device_set_version_format(FU_DEVICE(self), FWUPD_VERSION_FORMAT_QUAD);
 	fu_device_add_private_flag(FU_DEVICE(self), FU_DEVICE_PRIVATE_FLAG_EXPLICIT_ORDER);
 	fu_device_add_private_flag(FU_DEVICE(self), FU_DEVICE_PRIVATE_FLAG_USE_PROXY_FOR_OPEN);
+	fu_device_add_private_flag(FU_DEVICE(self), FU_DEVICE_PRIVATE_FLAG_ONLY_WAIT_FOR_REPLUG);
```

Comment:
I've retested couple times, summarized the impacted devices as following (which I think should add this flag):
* ec
* rts usbs
* rmm

The following devices have added this flag in their own plugin
* mst
* usb4

The following devices never show this error, as they are probed by ec dynamically:
* dpmux
* ilan
* pkg
* pd
* wtpd

Reply:
That looks pretty good; can you edit my original patch to only set the flag for ec, rts and rmm please.